tests/tests.go

```diff
@@ -42,6 +41,7 @@ func init() {
 	flag.StringVar(&falcoBinary, "falco-binary", falcoBinary, "Falco executable binary path")
 	flag.StringVar(&falcoctlBinary, "falcoctl-binary", falcoctlBinary, "falcoctl executable binary path")
 	flag.StringVar(&falco.FalcoConfig, "falco-config", falco.FalcoConfig, "Falco config file path")
+	flag.StringVar(&falco.FalcoContainerPluginLibrary, "falco-container-plugin", falco.FalcoContainerPluginLibrary, "Path to the Falco container plugin shared object.")
```

Comment:
I thought it could be helpful being able to run the testing framework pointing to a local libcontainer.so, to eventually test container plugin changes against the testing framework.
By default, it expects to find the shared object under `/usr/share/falco/plugins/libcontainer.so`.